plugins/modules/monitoring/icinga2_host.py

```diff
@@ -51,46 +53,61 @@
     type: bool
     default: 'no'
   client_cert:
+    type: path
     description:
       - PEM formatted certificate chain file to be used for SSL client
         authentication. This file can also include the key as well, and if
         the key is included, C(client_key) is not required.
   client_key:
+    type: path
     description:
       - PEM formatted file that contains your private key to be used for SSL
         client authentication. If C(client_cert) contains both the certificate
         and key, this option is not required.
   state:
+    type: str
     description:
       - Apply feature state.
     choices: [ "present", "absent" ]
     default: present
   name:
+    type: str
     description:
       - Name used to create / delete the host. This does not need to be the FQDN, but does needs to be unique.
     required: true
   zone:
+    type: str
     description:
       - The zone from where this host should be polled.
   template:
+    type: str
     description:
       - The template used to define the host.
       - Template cannot be modified after object creation.
   check_command:
+    type: str
     description:
       - The command used to check if the host is alive.
     default: "hostalive"
   display_name:
+    type: str
     description:
       - The name used to display the host.
-    default: if none is give it is the value of the <name> parameter
+      - If not specified, it defaults to the value of the I(name) parameter.
   ip:
+    type: str
     description:
       - The IP address of the host.
     required: true
   variables:
+    type: dict
     description:
       - List of variables.
+  force:
+    description:
+      - This option is parsed but not used by the module.
```

Comment:
```suggestion
```
Let's remove it so we don't have problems when ansible-core "suddenly" removes this option (after all it's deprecated). The option's description already says that it is deprecated and will be removed eventually, so I guess it's ok if it doesn't do anything :)

Reply:
works for me